ros-kinetic/mavros/Manifest

```diff
@@ -1,3 +1 @@
-DIST mavros-release-0.19.0.tar.gz 127873 SHA256 7a8520d6e05e7b423415ffb475d6f80ee081e8dd7e1110b60f5fec797e244aa4 SHA512 30abfb84e751ac1b682c302b5a2767f8176a71f3a02fc5b2369fd10d727aa26ae1d1dba5a93cf08545e5b9177c3fe6c580513715820cd7cb80651053c2e0ea2d WHIRLPOOL d7c0abba4b283eb157c17374d69a25377a4292acecda6a0d3ff77d7e356ffb814958d8e76e3e743cd004743fbd78948fdf4bd2f39296194b0d724ab57682ad8e
-EBUILD mavros-0.19.0.ebuild 1098 SHA256 f800e0c798edd40ff6ded82dabb2b65d5b6a7e6bee3dd84d61fc6ce98af98673 SHA512 820db1df6319b1b7bed337a8a2f76a6d88fdfc92eb3081269a78241ae6641156bbd57d850b672ab58244fde95883c246f9b12c6223216b514edcea3cb083fb58 WHIRLPOOL 48a967f8273fe9011f099bb211226a83e8e69f6ab6e5a91212100f2cc2368152611e7196d2bd71cfdd64b427847dfb1feca24246979ed7008f28bd4955be7ab1
```

Comment:
@tfoote Did the mavros package disappear? Not sure why the ebuild is being removed... I'm going to hold off on this one until I figure out why that happened...